src/metabase/util.clj

```diff
@@ -423,6 +423,21 @@
   (^String [s max-length]
    (str/join (take max-length (slugify s)))))
 
+(defn slugify-filename
+  "Return a version of String `s` appropriate for use as part of a file path.
+   Downcase the name, remove diacritcal marks, and replace non-alphanumeric *ASCII* characters with underscores;
+   Non-ASCII characters are preserved - modern filesystems support them.
```

Comment:
Sorry for the trivial nitpick but this is going to bother me and Emacs is going to try to correct it if I touch this which will lead to needless code churn 

https://guide.clojure.style/#docstring-indentation

Reply:
I realize now this is actually a result of copying `u/slugify`... if you see evil indentation please fix it

Reply:
Fixed.